ProQuest.js

```diff
@@ -219,23 +219,22 @@ function doWeb(doc, url, noFollow) {
 		});
 	}
 	else {
-		var abstractTab = doc.getElementById('tab-AbstractRecord-null') // Seems like that null is a bug and it might change at some point
-			|| doc.getElementById('tab-Record-null'); // Shown as Details
-		if (!(abstractTab && !abstractTab.classList.contains('active'))) {
-			Zotero.debug("On Abstract page, scraping");
+		var abstractTab = doc.getElementById('addFlashPageParameterformat_abstract');
+		if (abstractTab && abstractTab.classList.contains('active')) {
+			Zotero.debug("On Abstract tab and scraping");
 			scrape(doc, url, type);
 		}
-		else if (noFollow) {
+		else if (noFollow) { // when is this used? wouldn't it be better to throw an error instead of brute forcing?
```

Comment:
when is this used? wouldn't it be better to throw an error instead of brute forcing?

Reply:
Evidently to prevent infinite redirect loops: https://github.com/zotero/translators/commit/bb2a0b519ddba6043f145a1e677f3d1333f2154c

I think it's good to have in case the design changes slightly and the `abstractTab.classList.contains('active')` check no longer works. If you can remove the added comment, I think this is ready to merge.